lib/mpl_toolkits/axes_grid1/anchored_artists.py

```diff
@@ -1,43 +1,156 @@
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
 
+from matplotlib import docstring
 from matplotlib.externals import six
-
+from matplotlib.offsetbox import (AnchoredOffsetbox, AuxTransformBox,
+                                  DrawingArea, TextArea, VPacker)
 from matplotlib.patches import Rectangle, Ellipse
 
-import numpy as np
-
-from matplotlib.offsetbox import AnchoredOffsetbox, AuxTransformBox, VPacker,\
```

Comment:
I am 50/50 on restoring these unused imports.

On one hand it is bad to confuse the API like this by moving classes around between modules.

On the other hand, this _is_ an api break.

In either case :+1: on changing the example to not use this.